.test-infra/jenkins/job_PostCommit_CrossLanguageValidatesRunner_Dataflow.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+// This job runs the suite of ValidatesRunner tests against the Dataflow runner.
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_XVR_Dataflow',
+    'Run XVR_Dataflow PostCommit', 'Dataflow CrossLanguageValidatesRunner Tests', this) {
+      description('Runs the CrossLanguageValidatesRunner suite on the Dataflow runner.')
+
+      // Set common parameters.
+      commonJobProperties.setTopLevelMainJobProperties(delegate)
+
+      // Publish all test results to Jenkins
+      publishers {
+        archiveJunit('**/nosetests*.xml')
+      }
+
+      // Gradle goals for this job.
+      // TODO: We only support Python pipeline using Java external transforms at the moment.
+      // Enable testing for Java pipeline using Python external transforms when BEAM-11203
+      // is implemented.
+      steps {
+        shell('echo "*** RUN CROSS-LANGUAGE DATAFLOW PYTHON WITH JAVA EXTERNAL TRANSFORMS USING PYTHON 3.6 ***"')
+        gradle {
+          rootBuildScriptDir(commonJobProperties.checkoutDir)
+          tasks(':runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerPythonUsingJava')
+          commonJobProperties.setGradleSwitches(delegate)
+          switches('-PpythonVersion=3.6')
+        }
+        shell('echo "*** RUN CROSS-LANGUAGE DATAFLOW PYTHON WITH JAVA EXTERNAL TRANSFORMS USING PYTHON 3.8 ***"')
+        gradle {
+          rootBuildScriptDir(commonJobProperties.checkoutDir)
+          tasks(':runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerPythonUsingJava')
+          commonJobProperties.setGradleSwitches(delegate)
+          switches('-PpythonVersion=3.8')
+        }
+        shell('echo "*** RUN CROSS-LANGUAGE DATAFLOW PYTHON WITH JAVA SQL TRANSFORMS USING PYTHON 3.8 ***"')
```

Comment:
Can we add Kafka as well ?

Reply:
I think it's not necessary. Kafka xlang test is already a part of python postcommit tests: https://github.com/apache/beam/blob/master/sdks/python/test-suites/portable/common.gradle#L177

Reply:
That only runs Kafka on Flink though right?

and thank you for adding the SQL task :) it looks like it's working:
![image](https://user-images.githubusercontent.com/675055/99000877-cde3b200-24ee-11eb-98e2-17b0322f03bb.png)

Reply:
Oh, that's right. Good catch 😄
Will take a look if we can easily add Kafka xlang test too.

Reply:
We can't simply adding xlang Kafkaio test here since it needs a separate Kafka cluster setup on Dataflow (on Flink it uses locally launched Kafka cluster). Will merge without xlang Kafkaio test.